homework-6/homework-6/ViewControllers/CatalogTableViewCell.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  CarTableViewCell.swift
+//  homework-6
+//
+//  Created by Рустам Шайхинуров on 21.10.2022.
+//
+
+import UIKit
+
+class CatalogTableViewCell: UITableViewCell {
+	override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+		super.init(style: style, reuseIdentifier: reuseIdentifier)
+		
+		setup()
+	}
+	
+	required init?(coder: NSCoder) {
+		super.init(coder: coder)
+		
+		setup()
+	}
+	
+	private let carImage: UIImageView = .init()
+	private let modelLabel: UILabel = .init()
+	private let weightLabel: UILabel = .init()
+	private let priceLabel: UILabel = .init()
+	private let maxSpeedLabel: UILabel = .init()
+	private let fuelConsumptionPerKmLabel: UILabel = .init()
+	
+	public func set(car: Car) {
+		carImage.image = UIImage(systemName: "car.fill")
+		modelLabel.text = car.model
+		weightLabel.text = "Вес: \(car.weight) кг"
+		priceLabel.text = "Цена: \(car.price) ₽"
+		maxSpeedLabel.text = "Максимальная скорость: \(car.maxSpeed) км/ч"
+		fuelConsumptionPerKmLabel.text = "Расход топлива: \(car.fuelConsumptionPerKm)л. на километр"
+	}
+	
+	public func toString() -> String {
+		let information = """
+Вес: \(weightLabel.text ?? "null")
+Цена: \(priceLabel.text ?? "null")
+Максимальная скорость: \(maxSpeedLabel.text ?? "null")
+Расход топлива: \(fuelConsumptionPerKmLabel.text ?? "null")
+"""
+		return information
+	}
```

Comment:
Отступы нужно соблюдать даже в многострочных строках

```suggestion
	public func toString() -> String {
		let information = """
                    Вес: \(weightLabel.text ?? "null")
                    Цена: \(priceLabel.text ?? "null")
                    Максимальная скорость: \(maxSpeedLabel.text ?? "null")
                    Расход топлива: \(fuelConsumptionPerKmLabel.text ?? "null")
                    """
		return information
	}
```